src/obu.rs

```diff
@@ -2362,18 +2320,17 @@ unsafe fn parse_obus(
                 // There's no trailing bit at the end to skip,
                 // but we do need to align to the next byte.
                 gb.bytealign();
-                parse_tile_grp(c, r#in, props, &mut gb, init_bit_pos, init_byte_pos, len)?;
+                parse_tile_grp(c, r#in, props, &mut gb)?;
```

Comment:
```suggestion
                parse_tile_grp(c, r#in, props, gb)?;
```

Now that `gb: &mut GetBits` instead of `gb: GetBits`, we can pass just `gb` instead of `&mut gb` or `&gb` everywhere we do in `fn parse_obus`.